packages/menu-surface/index.tsx

```diff
@@ -368,7 +356,8 @@ class MenuSurface extends React.Component<MenuSurfaceProps, MenuSurfaceState> {
         {...otherProps}
       >
         {children}
-      </div>
+      </div>,
+      document.body
```

Comment:
are you familiar with SSR? Will this break SSR?

Reply:
I've not worked with SSR but from my understanding it doesn't support portals. Does menu-surface currently support SSR? The simple approach would be check if document exists and return null if it doesn't. I'll add a commit which does this.

Reply:
Cool that sounds good...Currently we have no SSR support, but we have been getting bugs - so I want to be cognizant of that.

I want to add SSR testing to the roadmap.